docker/test/integration/minifi/test/__init__.py

```diff
@@ -142,7 +147,7 @@ def check_output(self, timeout=5):
         self.wait_for_output(timeout)
         self.log_nifi_output()
 
-        return self.output_validator.validate()
+        return self.output_validator.validate() & ~self.segfault
```

Comment:
What's the goal here? The left argument is a boolean.

Reply:
I'm not a python developer ( first time were recent PRs ) -- after doing some reading I'm guessing the boolean operators are preferred here? It works due to autoboxing as it's a very simple expression; however, your statement "left argument is a boolean" implies that it is not common to use the bitwise operators here?

Reply:
I have hardly seen, but the question was meant to ask what do we plan to achieve here. 

I just run through the code to understand what's going on here, this if fine, but using logical expressions are way more _pythonic_ to achieve the same:

```
return self.output_validator.validate() and not self.segfault
```

Reply:
I would much rather learn and do what is more generally appropriate for that language domain. Thanks!